internal/consumer/service.go

```diff
@@ -0,0 +1,80 @@
+package consumer
+
+import (
+	"log"
+	"sync"
+
+	"github.com/pkg/errors"
+)
+
+type Item struct {
+	By          string `bson:"by"`
+	Title       string `bson:"title"`
+	Url         string `bson:"url"`
+	Text        string `bson:"text"`
+	ItemType    string `bson:"itemType" json:"type"`
+	Descendants int    `bson:"descendants"`
+	Id          int    `bson:"id"`
+	Score       int    `bson:"score"`
+	Time        int    `bson:"time"`
+	Parent      int    `bson:"parent"`
+	Poll        int    `bson:"poll"`
+	Kids        []int  `bson:"kids"`
+	Parts       []int  `bson:"parts"`
+	Deleted     bool   `bson:"deleted"`
+	Dead        bool   `bson:"dead"`
+}
+
+// Execute - Entry point for consumer service
+func Execute(dbRepo DbRepository, dataService DataService) {
```

Comment:
I'm not a big fan of some of these names e.g. dbRepo, DbRepository.
I would prefer to use the full name like you did dataService to be clearer